examples/Demo/FluentUI.Demo.Client/Documentation/Components/Menu/FluentMenu.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Menu
+route: /Menu
+---
+
+# Menu
+
+A menu displays a list of actions and can be attached to a button or any other element. The element the menu is attached to must have an `id` attribute
+set. This `id` needs to be passed to the `FluentMenu` component in the `Trigger` parameter. The menu is then displayed when the element is clicked.
+
+In total the menu consists of three components:
+- `FluentMenu`: This is the base component which encapsulates all menu functionality.
+As described above, you use the `Trigger` parameter to connect the menu to an element that
+triggers showing or hiding the menu.
+- `FluentMenuList`: This component contains the list of menu items that needs to be shown.
+- `FluentMenuItem`: This component represents a single menu item. You can add any content to the menu
+item by using either the `ChildContent` parameter or the `Label` parameter. (If you use both, both
+will be displayed to the end user). You can create submenus by using the `MenuItem` parameter and adding new `FluentMenuItems` in there. 
+
+See the API documentation below for a complete overview of the parameters and event handlers.
+
+### Best Practices
```

Comment:
Can you reduce the header level => `##`.
If not, the "On this page" section on the right cannot be displayed.